### PR TITLE
[ETCM-432] Disable switching UI to erro state after encountering erro…

### PR DESCRIPTION
**Pull request by kapke**
…r when refreshing state, remove WalletError state and UI components

### CODE REVIEW

**Review by conqeror — APPROVED** [review]
Btw, I find quite confusing that `refreshSyncStatus` does a lot of stuff other than only refreshing sync status (it actually refreshes pretty much the whole wallet state). Shoudn't we rather have load function (same as for transaction history, balance, ...) for sync status and then `refreshWalletState` just calling `load` function and handling errors?

**Comment by kapke** [comment]
TBH I find it confusing too. That's going to be changed to some extent at #20 as transaction history runs independently from other pieces there.

File: src/wallets/Wallets.tsx [hunk 1]
```diff
@@ -17,6 +20,27 @@ export const Wallets = (): JSX.Element => {
...
+      option.map((error) => translateError(error)),
+      option.fold(
+        () => void 0,
+        (error) => {
+          message.error(error, 10, () => {
+            switch (walletState.walletStatus) {
+              case 'INITIAL':
+              case 'LOADED':
+              case 'LOADING':
+                walletState.refreshSyncStatus()
```
Comment (conqeror) [inline src/wallets/Wallets.tsx:36]: Should we provide the refresh function also here when it's always refreshing in the background anyway?

**Review by conqeror — APPROVED** [review]

File: src/wallets/Wallets.tsx (hunk 1 above)
Comment (kapke) [reply src/wallets/Wallets.tsx:36]: Refreshing was stopping on error and was stuck on `loading` state if refresh wasn't triggered here

Comment (kapke) [reply src/wallets/Wallets.tsx:36]: At least if the error happened during the initial load

Comment (kapke) [reply src/wallets/Wallets.tsx:36]: After changes from #48 It should work quite nicely together I think

Comment (conqeror) [reply src/wallets/Wallets.tsx:36]: yeah, that's also what I'm thinking. It's coming together :D

**Pull request merged**